Formula/v8.rb

```diff
@@ -1,10 +1,10 @@
-# Track Chrome stable.
-# https://omahaproxy.appspot.com/
+# Track Chrome stable, see https://omahaproxy.appspot.com/
 class V8 < Formula
   desc "Google's JavaScript engine"
   homepage "https://github.com/v8/v8/wiki"
-  url "https://github.com/v8/v8-git-mirror/archive/5.1.281.47.tar.gz"
-  sha256 "63c9933227d6912689ea6bc012eea6a1fabaf526ac04bc245d9381e3ea238bf6"
+  url "https://chromium.googlesource.com/chromium/tools/depot_tools.git",
+    :revision => "d61997d37c8f7cc68080f39d4b516fb7b269b46c"
+  version "6.8.275.26"
```

Comment:
Wondering if we should add a comment next to this with something like `# v8 version, not depot_tools version`. Maybe it's clear enough already. Unsure.